foam_controlwrapper/examples/poiseuille_parallel.py

```diff
@@ -0,0 +1,62 @@
+"""Example to solve 2D poiseuille flow
+
+"""
+
+from simphony.core.cuba import CUBA
+from simphony.engine import openfoamII
```

Comment:
I do not see any sign of `openfoamII`.  Is the related work missing or is it still a work in progress?